src/unix/device.c

```diff
@@ -0,0 +1,97 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Please set the copyright to "The libuv project and contributors." Here and in other files you added.